src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/DuplexClientBaseTests.cs

```diff
@@ -15,10 +13,45 @@ public static class DuplexClientBaseTests
 
     [Fact]
     [OuterLoop]
+    public static void DuplexClientBaseOfT_OverHttp_Call_Throws_InvalidOperation()
+    {
+        DuplexClientBase<IWcfDuplexService> duplexService = null;
+        Guid guid = Guid.NewGuid();
+
+        try
+        {
+            BasicHttpBinding binding = new BasicHttpBinding(BasicHttpSecurityMode.None); 
+
+            WcfDuplexServiceCallback callbackService = new WcfDuplexServiceCallback();
+            InstanceContext context = new InstanceContext(callbackService);
+
+            duplexService = new MyDuplexClientBase<IWcfDuplexService>(context, binding, new EndpointAddress(Endpoints.Https_DefaultBinding_Address));
+            Assert.Equal(CommunicationState.Created, duplexService.State);
+
+            var exception = Assert.Throws<InvalidOperationException>(() => 
+            { 
+                IWcfDuplexService proxy = duplexService.ChannelFactory.CreateChannel();
+            });
+
+            // Can't compare the entire exception message - .NET Native doesn't necessarily output the entire message, just params
+            // "Contract requires Duplex, but Binding 'BasicHttpBinding' doesn't support it or isn't configured properly to support it"
+            Assert.True(exception.Message.Contains("BasicHttpBinding")); 
+        }
+        finally
+        {
+            if (duplexService != null && duplexService.State != CommunicationState.Closed)
+            {
+                duplexService.Abort();
+                Assert.Equal(CommunicationState.Closed, duplexService.State);
+            }
```

Comment:
I will, however, leave `DuplexClientBaseOfT_OverNetTcp_Call` to fulfill the purpose of the "normal path" test, including the state.